### PR TITLE
chore: deprecates Include config field.

### PR DESCRIPTION
**Pull request by jcchavezs**
It can be done in 'directives' field and only causes confusion with respect to ordering of rules which is something that matters when it comes to disruptive actions.

### CODE REVIEW

File: coraza.go [hunk 1]
```diff
@@ -54,8 +55,8 @@ func (m *corazaModule) Provision(ctx caddy.Context) error {
 		config = config.WithDirectives(m.Directives)
 	}
 
-	m.logger.Debug("Preparing to include files", zap.Int("count", len(m.Include)), zap.Strings("files", m.Include))
 	if len(m.Include) > 0 {
+		m.logger.Warn("Include field is deprected, do the include in directives instead")
```
Comment (fzipi) [inline coraza.go:59]:
```suggestion
		m.logger.Warn("include field is deprecated, please use the Include directive instead")
```

**Review by fzipi — COMMENTED** [review]
There in no `Include` in seclang. Are we extending it?

**Comment by jcchavezs** [comment]
Maybe the wording isn't the best one bue Include is a known directive in
modsecurity/coraza (maybe it isn't part of seclang and it is more an Apache
thing)

**Comment by fzipi** [comment]
Then let's change the wording so we don't confuse users. With this in mind, the integration has the requirement of providing this feature, or things won't work as expected...

**Comment by jcchavezs** [comment]
I don't think we need to provide this. Whenever you use coraza you are used
to do `Include @owasp_crs/*.conf` in the directives isn't? We expect the
same here.

**Comment by fzipi** [comment]
Ok, now I get it (I thought we were discussing coraza, instead of the connector 🤦).

**Review by fzipi — APPROVED** [review]

File: coraza.go [hunk 2]
```diff
@@ -54,8 +55,8 @@ func (m *corazaModule) Provision(ctx caddy.Context) error {
 		config = config.WithDirectives(m.Directives)
 	}
 
-	m.logger.Debug("Preparing to include files", zap.Int("count", len(m.Include)), zap.Strings("files", m.Include))
 	if len(m.Include) > 0 {
+		m.logger.Warn("include field is deprecated, please use the Include directive instead")
```
Comment (jcchavezs) [inline coraza.go:59]:
```suggestion
		m.logger.Warn("include field is deprecated, please use the Include directive inside 'directives' field instead")
```

**Pull request merged**